apm.lock.yaml

```
@@ -0,0 +1,14 @@
+lockfile_version: '1'
```

YAMLlint (1.38.0)

[error] wrong new line character: expected \n

(new-lines)